modules/person.rb

```diff
@@ -1,7 +1,7 @@
 require_relative 'nameable'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ]  Kindly make `rentals`  to have Getters and Setters. Kindly make the modification to meet the condition of a person has many rentals.